Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM python:3.10
```

Comment:
**suggestion**: you could get a smaller image using a different distribution. But you will need to modify your instructions to help make it build. It's a trade off but probably worth it.

Reply:
I'm interested. What would the advantages be?

Reply:
Smaller image size. Less cve's.